content/docs/communities/moderation/_index.md

```diff
@@ -95,21 +93,280 @@ in the message bar at the bottom:
+your community.
+
+If you have reports enabled, and click on the 
+{{ mjolnirbutton(text="🚫 Ban") }} button below it, mjolnir will ban the user
+from all the rooms it protects. The user will be banned not only from the room
+where they were reported, but from your whole community.
+
+If you don't have moderation reports enabled, you can also ban either a user
+with the following command
+
+```
+!mjolnir ban <list shortcode> <user|room|server> <glob> [reason]
+```
+
+For example, to remove `@john:example.com` permanently from your community for
+infringement to the code of conduct, you can issue the following command in
+mjolnir's control room.
+
+```
+!mjolnir ban coc user @john:example.com User keeps insulting people
+```
+
+That's a complex command, let's try to understand what it does.
+
+- `!mjolnir ban` tells mjolnir we want it to perform a ban
+- `coc` tells mjolnir we want to add the person to the ban list with shortname
+  `coc`. We'll explore what ban lists are a little further below.
+- `user` tells mjolnir it needs to ban a user. It is not strictly mandatory, but
+   it's a good practice to add it in order to avoid banning a whole server when
+   trying to ban a single user, in case of typo.
+- `@john:example.com` is the Matrix ID of the person we want to ban
+- `User keeps insulting people` is the reason why we are banning them. Their
+   client will be able to display that message.
+
+Mjolnir then adds the user we're banning to the ban list we specified. Mjolnir
+keeps watching one or several ban lists, and will ban all the users present on
+those lists from all the rooms it protects.
+
+Note that this command will likely fail if it's the first time you are trying to
+ban someone. Indeed, you need to create one or more ban lists before adding
+users or servers on them. See the section [Creating ban lists](#creating-ban-lists)
+below for more information on how to create them.
+
+### Removing a server definitely
+
+Matrix is a federated network. This means people can set up new servers
+specifically to harm some communities. This allows them to create a virtually
+infinite number of users, making it possible to raid other communities easily.
+
+Fortunately, all these malicious users have one thing in common: they come from
+the same homeserver. This means if you block the malicious homeserver entirely,
+all the malicious users are going to be banned at once.
+
+To do so, you can issue a variant of the command used to ban an individual user.
+The command was
+
+```
+!mjolnir ban <list shortcode> <user|room|server> <glob> [reason]
+```
+
+As you probably guessed, you are not going to ban a user but a server this once.
+For example, to ban the `maliciousdomain.tld` domain:
+
+```
+!mjolnir ban spam server maliciousdomain.tld
+```
+
+Mjolnir is going to add the domain to the `spam` ban list, and whenever someone
+from that homeserver tries to join your community they are going to be banned.
+
```

Comment:
Missing is banning/disabling rooms for server admins. Or I guess in general the Synapse plugin is missing.

Reply:
I'm not sure I understand this suggestion :)